test/Unit/Common.hs

```diff
@@ -49,10 +53,10 @@ testCurrentAuctionStage = do
 
   let config =
         AuctionTermsConfig
-          { configDiffBiddingStart = 1 * 60 - 1
-          , configDiffBiddingEnd = 2 * 60 - 1
-          , configDiffVoucherExpiry = 3 * 60 - 1
-          , configDiffCleanup = 4 * 60 - 1
+          { configDiffBiddingStart = 1 * 60 - 2
+          , configDiffBiddingEnd = 2 * 60 - 2
+          , configDiffVoucherExpiry = 3 * 60 - 2
+          , configDiffCleanup = 4 * 60 - 2
```

Comment:
Perhaps it can be documented why we choose to have these slightly weird times.

Reply:
Which part (or both) do you think we should explain better?